Source/ProjectRimFactory/Common/PRFGameComponent.cs

```diff
@@ -80,14 +93,49 @@ public static void RegisterAssemblerQueue(IAssemblerQueue queue)
             }
             return false;
         }
-        /*
-        // A quick way to test all the scupltures available, if need be.       
-        public override void LoadedGame() {
-            base.LoadedGame();
-            foreach (var t in Current.Game.CurrentMap.spawnedThings.Where(x => x is Building_Art)) {
-                if (!Current.Game.GetComponent<PRFGameComponent>().TryAddSpecialSculpture(t)) return;
-                Log.Warning("---------added test special scuplture: " + t + " at " + t.Position);
+
+        public override void LoadedGame()
```

Comment:
This gets called after `SpawnSetup` therefor i cant call `AssemblerQueue.Clear`
is there something that we could use instead (called before `SpawnSetup`)

Since the data in a `GameComponent` exists until the user quits to the OS we need to clear the Queue between different saves.
something OnLoad but before SpawnSetup of individual things would be perfect _(other things could then also be simplified)_

Reply:
I thought GameComponents were created on a new load and only exist while a game is being played?

AssemblerQueue should be empty when first created (as I understand it) - is this not true?

Reply:
Oh.  Make it non-static.  There's no benefit to having it static, is there?  Accessing it quickly can be done by having a static variable that points to the actual (local) queue (that is set in the game component constructor) if you need it to be fast.

Reply:
As I noted (in possibly another place; I don't know github that well), you can probably use a local field instead of a `static` field, and if you need fast access have a static member that points to it

Reply:
hmm during my tests it looked like `GameComponents` are created when game starts independent from Saves & Loads.
but i will try without static